doc/content/reference/telemetry/identity_server.md

```diff
@@ -0,0 +1,102 @@
+---
+title: "Identity Server Telemetry"
+description: "Description of everything that the telemetry procedure collects regarding the IdentityServer"
+aliases: "/telemetry/identity-server"
+---
+
+[The Things Stack]({{<ref "/the-things-stack">}})'s Identity Server collects telemetry data, which is used to help us understand how users interact with the product. For example, the current data helps to have a better insight on the user's usage of organizations. The objective with the introduction of telemetry is to be able to better understand what is used so that we can expand and improve on the experience that we provide to our users.
+
+## Identification
+
+The worry that data is attached to an easily identifiable piece of information is common and for good reason, therefore we believe that it is the first thing we should address.
+
+When the Identity Server starts the background task responsible for collecting and sending telemetry data, it generates the UID which is attached to every telemetry message that is sent, its value is nothing more than the base64 enconding of `console.ui[IS,GS,NS,AS,JS].BaseURL` values.
+
+
+The code bellow shows the exact procedure and it can also be seen on the open-source repository [here](https://github.com/TheThingsNetwork/lorawan-stack/blob/c8e625243ba8106a5240d4c5d7b0feb0fc1e9a3c/pkg/telemetry/exporter/hash.go#L23-L30).
+```go
+// GenerateHash receives a string slice and returns a string of an UID.
+func GenerateHash(_ context.Context, elems ...string) string {
+	s := sha512.New()
+	for _, elem := range elems {
+		s.Write([]byte(elem + ":"))
+	}
+	return base64.StdEncoding.EncodeToString(s.Sum(nil))
+}
+```
+
+{{< note >}}
+If you are running the enterprise version, the license ID is also included in the hash in order to guarantee uniqueness.
+{{</ note >}}
+
+
+## Types of telemetry data
+
+Bellow is an JSON object with an example of a full telemetry data message that is sent from the Identity Server to us:
+
+```json
+{
+  "message": {
+    "uid": "m64Jv5T3Bqnxz3tSgOAYD0w25UqWwP5DgIsuKbS0g5dBQkA8ZEHwlV1iDoKgjeF7uE3dt+3SDMdANJ41xDZqag==",
+    "os": {
+      "operating_system": "linux",
+      "arch": "amd64",
+      "binary_version": "3.27.0-dev",
+      "golang_version": " go1.20.7"
+    },
+    "cli": null,
+    "entities_count": {
+      "gateways": {
+        "total": 0,
+        "gateways_by_frequency_plan_id": {}
```

Comment:
This should have a small example of contents, otherwise people may interpret it as us providing gateway IDs/names to the collection server.